python/taichi/lang/matrix.py

```diff
@@ -17,7 +17,7 @@
                               taichi_scope, to_numpy_type, to_paddle_type,
                               to_pytorch_type, warning)
 from taichi.types import primitive_types
-from taichi.types.compound_types import CompoundType
+from taichi.types.compound_types import CompoundType, TensorType
```

Comment:
Yea `TensorType` is confusing for DL users but currently users don't directly manipulate this class yet so I'll go for this for now. (Mainly because we `MatrixType` name is already taken lol ....